site/app/views/UserProfileView.php

```diff
@@ -62,4 +62,4 @@ public function showUserProfile(
             'user_utc_offset' => $user_utc_offset
         ]);
     }
-}
+}
```

Comment:
The whitespace changes should be removed - I think our linters should fix the newlines at the end of file issue

see 

https://submitty.org/developer/testing/linting_static_analysis#python-linting and 

https://submitty.org/developer/testing/linting_static_analysis#php-linting